docs/application/dotnet/guides/nui/visuals.md

```diff
@@ -400,177 +575,198 @@ The following features are not supported:
 
-**Figure: Primitive visual**
+<!---------------------------------------------------------------------------------------------------------------------------->
+<a name="meshVisual"></a>
+### MeshVisual
 
-![Primitive visual](media/cube.png)
+The mesh visual renders a mesh using an `.obj` file, optionally with materials provided in an `.mtl` file and corresponding textures. 
 
-The following table lists the supported properties. The visual type is `Primitive` and the visual map is `PrimitiveVisual`.
 
-**Table: Primitive visual properties**
+The following table lists the supported properties:
 
-| Property            | Type              | Description                              |
-|-------------------|-----------------|----------------------------------------|
-| `Shape`             | `integer` or `string` | The shape to render.          |
-| `MixColor`          | `Vector4`           | The color of the shape.                |
-| `Slices`            | `integer`           | The number of slices going around the shape. |
-| `Stacks`            | `integer`           | The number of layers going down the shape. |
-| `ScaleTopRadius`    | `float`             | For conical frustums, the top circle scale radius. |
-| `ScaleBottomRadius` | `float`             | For conical frustums, the bottom circle scale radius. |
-| `ScaleHeight`       | `float`             | For conics, the scale height.    |
-| `ScaleRadius`       | `float`             | For cylinders, the scale radius.  |
-| `ScaleDimensions`   | `Vector3`           | For cuboids, the dimensions. They scale in the same way as for a 9-patch image. |
-| `BevelPercentage`   | `float`             |  For cuboids, the amount of beveling is based on the cuboid's smallest dimension.|
-| `BevelSmoothness`   | `float`             |  For cuboids, the beveled edge smoothness. |
-| `LightPosition`     | `Vector3`           | The position, in the stage space, of the point light, that apply lighting to the model. |
+**Table: MeshVisual properties**
+
+| Property         | Type                | Required              | Description                              |
+|------------------|---------------------|-----------------------|------------------------------------------|
+| `ObjectURL`      | `string`            | Yes                   | The location of the `.obj` file.         |
+| `MaterialtURL`   | `string`            | No                    | The location of the `.mtl` file. Leave blank for a textureless object. |
+| `TexturesPath`   | `string`            | Yes if using materials| The path to the directory where the textures (including gloss and normal) are stored. |
+| `ShadingMode`    | `MeshVisualShadingModeValue` | No           | The type of the shading mode that the mesh uses (see tab. [with possible values](#table_mesh_possible_values)). |
```

Comment:
```suggestion 
| `ShadingMode`    | `MeshVisualShadingModeValue` | No           | The type of the shading mode that the mesh uses. For more information, see [possible values](#table_mesh_possible_values). |

Reply:
I am waiting for your reply to the https://github.com/Samsung/tizen-docs/pull/1367#discussion_r537940183 - I will use the same scheme everywhere.
currently I write like this:

The type of the shading mode that the mesh uses (see the [below table](#table_mesh_possible_values) for more information)